### DISCUSSION TITLE
I can't find Neuralnet in opentrack

### DISCUSSION THREAD

**Discussion by KPgreg07** [opentrack/opentrack#1910] [Q&A]
Hello,
I recently posted something similar in issues, but I didn't notice that there is an area for discussions. So, when I installed opentrack 2023.3.0, I cannot find neuralnet tracking at all in my inputs catalogue. Could i have possibly installed opentrack incorrectly? Do I need to install the zip file instead? Am I just dumb?

**Comment by Blogames** [comment]
![image](https://github.com/user-attachments/assets/70135194-a461-4bb0-b07f-ca8e905872fc)
It's there for me, maybe you just don't see it, if you don't see it then just reinstall.

**Reply by KPgreg07** [reply]
I've uninstalled and re-installed about four times, but it's worth another shot!! Also, did you have to install another program before? Someone said to try ONNX Runtime on Windows, and I'm at a complete loss on how to install/run it!

**Reply by Blogames** [reply]
> Also, did you have to install another program before? 

No.

Well, if you want, you could use the though outdated but still working [AITrack](https://github.com/AIRLegend/aitrack/releases/tag/v0.7.1-alpha) with UDP over network.

**Reply by DaWelter** [reply]
> I've uninstalled and re-installed about four times, but it's worth another shot!! Also, did you have to install another program before? Someone said to try ONNX Runtime on Windows, and I'm at a complete loss on how to install/run it!

The `onnxruntime.dll` and everything else should readily come with opentrack.

My experience with install problems is limited. So here are wild guesses: Issues with the MSVC runtime. Or overeager anti-virus software.

You could also try to run opentrack in cmd to see an error message maybe.

**Comment by Cory-Watson** [comment]
I'm having the same problem with both 2023.3.0 and 2023.2.0.

If I launch the binary in CMD I get a DEBUG log about a missing module, but it looks like it is looking in the wrong disk. It's trying to find it on my F: disk instead of C: where it is installed. I'm guessing this may be why the NeuralNet option isn't available..?

```
DEBUG [F:\dev\opentrack\api/plugin-support.hpp:106]: library "kinect-face" failed: "Cannot load library C:\\Program Files (x86)\\opentrack\\modules\\\\opentrack-tracker-kinect-face.dll: The specified module could not be found."
```

**Comment by DaWelter** [comment]
@Cory-Watson, `F:\dev\opentrack\api/plugin-support.hpp:106` Is just the source code location where the error was logged.
Failing to load `opentrack-tracker-kinect-face.dll`? @sthalik , didn't you say such issues can be caused by anti virus software?

**Reply by sthalik** [reply]
Install the Kinect SDK. Likely you haven't built the module yet if the error hasn't appeared previously, or are now using the compressed SDK tree without the installer.
